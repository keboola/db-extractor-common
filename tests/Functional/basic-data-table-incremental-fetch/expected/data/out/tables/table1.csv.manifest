{"destination":"table1","incremental":false,"metadata":[{"key":"KBC.name","value":"table1"},{"key":"KBC.sanitizedName","value":"table1"},{"key":"KBC.schema","value":"testdb"},{"key":"KBC.type","value":"BASE TABLE"},{"key":"KBC.rowCount","value":"2"},{"key":"KBC.autoIncrement","value":"3"}],"column_metadata":{"id":[{"key":"KBC.datatype.type","value":"int"},{"key":"KBC.datatype.nullable","value":false},{"key":"KBC.datatype.basetype","value":"INTEGER"},{"key":"KBC.datatype.length","value":"10"},{"key":"KBC.sourceName","value":"id"},{"key":"KBC.sanitizedName","value":"id"},{"key":"KBC.primaryKey","value":true},{"key":"KBC.ordinalPosition","value":"1"},{"key":"KBC.constraintName","value":"PRIMARY"},{"key":"KBC.extra","value":"auto_increment"},{"key":"KBC.autoIncrement","value":"3"}],"col1":[{"key":"KBC.datatype.type","value":"varchar"},{"key":"KBC.datatype.nullable","value":false},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"128"},{"key":"KBC.sourceName","value":"col1"},{"key":"KBC.sanitizedName","value":"col1"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.ordinalPosition","value":"2"}],"col2":[{"key":"KBC.datatype.type","value":"varchar"},{"key":"KBC.datatype.nullable","value":false},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"128"},{"key":"KBC.sourceName","value":"col2"},{"key":"KBC.sanitizedName","value":"col2"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.ordinalPosition","value":"3"}]},"columns":["id","col1","col2"]}
