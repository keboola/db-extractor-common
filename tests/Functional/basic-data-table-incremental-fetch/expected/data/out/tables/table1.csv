"1","row1","r1"
"2","row2","r2"
